src/Map.jsx

```diff
@@ -103,7 +104,37 @@ class Map extends Component {
       { force: true },
     );
   }
-
+  /**
+   * return coordinates in an array [lon, lat]
+   */
+  getCoordinateArray(stop) {
+    console.log(this);
+    const stopPoints = [];
+    stopPoints.push(stop.lat);
+    stopPoints.push(stop.lon);
+    return stopPoints;
+  }
+  /**
+   * given the two selected stop sids, returns a line segment
+   * between them
+   */
+  getRouteBetweenStops(routeStops) {
+    const stopSids = this.state.selectedStops.map(stop => stop.sid);
+    stopSids.sort((a, b) => a - b);
```

Comment:
if you're trying to sort descending, would `sort().reverse()` look cleaner?

Reply:
@EddyIonescu JavaScript sorting converts each element into a string before it sorts.
So the override that I've added is for sorting numbers